packages/gasket-nextjs/package.json

```diff
@@ -43,7 +43,9 @@
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-nextjs",
   "dependencies": {
     "@gasket/data": "^6.41.1",
-    "htmlescape": "^1.1.1"
+    "htmlescape": "^1.1.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0"
```

Comment:
Why did these get moved?

Reply:
Great catch! This actually needed to become move to `peerDependencies`. Thanks for the review!